python/raft-ann-bench/src/raft-ann-bench/plot/__main__.py

```diff
@@ -397,6 +429,15 @@ def main():
                               algorithms",
         default=None,
     )
+    parser.add_argument(
+        "--groups",
+        help="plot only comma separated groups of parameters",
+        default="base"
+    )
+    parser.add_argument(
+        "--algo-groups",
+        help="add comma separated algorithm+groups to the plot",
+    )
```

Comment:
Probably work providing an example of this (using groups that are already specified.) Something like  "raft_ivf_pq.large,hnswlib.large"